tests/gui/models/supported-formats.spec.js

```diff
@@ -62,62 +62,80 @@ describe('Browser: SupportedFormats', function() {
     describe('.isSupportedImage()', function() {
 
       it('should return false if the file has no extension', function() {
-        const isSupported = SupportedFormatsModel.isSupportedImage('/path/to/foo');
+        const isSupported = SupportedFormatsModel.isSupportedImage({
+          path: '/path/to/foo'
+        });
```

Comment:
Rather than creating lots of objects with just a `path` property, perhaps it'd be better to have a 'proper' image object, which all the unit-tests would use, just changing the `path` property each time?